src/record/export/usecases/get.ts

```diff
@@ -44,6 +48,11 @@ export const getRecords = async (
           attachmentsDir,
         }),
     );
+    if (localRecords.length === 0) {
```

Comment:
MUST:
Because `localRecords` can be `[]` when the number of target records is an integer multiple of GET_RECORDS_LIMIT, this condition is not correct.
example: The GET_RECORDS_LIMIT is 500, so when I export 1000 records, the warning message is shown after all records are exported.
So, please count how many records we proceeded and show a warning if the number of total proceeded records is zero.


---
For future refactoring:

Currently, `getAllRecordsFn()` returns `{done: false, value: []}` when no target records exist.
It's better to make `getAllRecordsFn()` returns `{done: true, value}`, then statements in `for await` statement aren't executed.
https://github.com/microsoft/TypeScript/blob/6cc605fcfe66118eafb50b2fdbd386da7e419795/src/lib/es2015.iterable.d.ts#L16